test/project_types/php/commands/create_test.rb

```diff
@@ -33,8 +33,8 @@ class CreateTest < MiniTest::Test
       APPTYPE
 
       def test_prints_help_with_no_name_argument
-        io = capture_io { run_cmd("create php --help") }
-        assert_match(CLI::UI.fmt(PHP::Commands::Create.help), io.join)
+        io = capture_io { run_cmd("php create --help") }
+        assert_match(CLI::UI.fmt(PHP::Command::Create.help), io.join)
```

Comment:
There's another method that could be used here, that's been added to `minitest_ext.rb`.  More useful for multi-line output comparisons, so might be a bit overkill for this.

```suggestion
        assert_message_output( io: io, expected_content: [PHP::Command::Create.help] )
```